src/ir_Mitsubishi.cpp

```diff
@@ -480,6 +480,16 @@ uint8_t IRMitsubishiAC::getWideVane(void) const {
   return _.WideVane;
 }
 
+/// Set the requested Left Vane (Vertical Swing) operation mode of the a/c unit.
```

Comment:
Is it "Vertical Swing" or "Horizontal Swing"? Your description `Mitsubishi144Protocol` (`VaneLeft`) indicates it's a "SwingH(Left)". Hence the question.

Reply:
it's Vertical Swing. Corrected the comment.